nautobot/docs/user-guide/platform-functionality/approval-workflow.md

```diff
@@ -1 +1,345 @@
+    - **Approval Workflow** panel:
+        - **Approval Workflow Definition** - Linked definition for the workflow.
+        - **Object Under Review** - Object subject to approval.
+        - **Current State** - Workflow status.
+        - **Approval Date** - Date of final decision.
+        - **Requesting User** - User who initiated the workflow.
+    - **Stages** panel:
+        - **Stage** - Stage name.
+        - **Actions Needed** - Remaining approvals required.
+        - **State** - Current stage status.
+        - **Decision Date** - Date of stage decision.
+    - **Responses** panel:
+        - **Stage** - Stage name.
+        - **User** - Responding user.
+        - **Comments** - Submitted comment.
+        - **State** - Decision state (`Pending`, `Approved`, `Denied`).
+
+    ![Approval Workflow details](../../media/development/approval-workflows/approval-workflow-details.png)
+
+### Using Approval Workflow via API
+
+#### Approve/Deny a Stage
+
+```no-highlight
+curl -X POST \
+-H "Authorization: Token $TOKEN" \
+-H "Content-Type: application/json" \
+-H "Accept: application/json; version=1.3; indent=4" \
+-d '{"comment": "Approved for deployment"}' \
+http://nautobot/api/extras/approval-workflow-stages/$APPROVAL_WORKFLOW_STAGE_ID/approve
+```
+
+```no-highlight
+curl -X POST \
+-H "Authorization: Token $TOKEN" \
+-H "Content-Type: application/json" \
+-H "Accept: application/json; version=1.3; indent=4" \
+-d '{"comment": "Deny reason"}' \
+http://nautobot/api/extras/approval-workflow-stages/$APPROVAL_WORKFLOW_STAGE_ID/deny
+```
+
+#### List Pending/Done Approvals
+
+Retrieves a list of approval workflow stages filtered by their status relative to the current user using the `pending_my_approvals` query parameter on the standard list endpoint:
+
+- `?pending_my_approvals=true` — Returns stages pending approval by the current user.
+
+```no-highlight
+curl -X GET \
+-H "Authorization: Token $TOKEN" \
+-H "Accept: application/json; version=1.3; indent=4" \
+http://nautobot/api/extras/approval-workflow-stages/?pending_my_approvals=true
+```
+
+- `?pending_my_approvals=false` — Returns stages the current user has already approved/denied.
+
+```no-highlight
+curl -X GET \
+-H "Authorization: Token $TOKEN" \
+-H "Accept: application/json; version=1.3; indent=4" \
+http://nautobot/api/extras/approval-workflow-stages/?pending_my_approvals=false
+```
+
+If the parameter is omitted, all stages are returned regardless of approval status.
+
+## Permissions by Persona
+
+### Object Operator
+
+Responsible for creating, updating, or running objects, which may trigger an approval workflow. Cannot approve workflows.
+
+**Required permissions:**
+- `extras.view_approvalworkflow` - to view created approval workflows after creating, updating, or running an object.
+- `extras.view_approvalworkflowstage` - to view the **My Requests** tab on the Dashboard.
+- `extras.view_approvalworkflowstageresponse` - to view responses for each stage.
```

Comment:
Not rendering correctly, I think this will fix it:

```suggestion
**Required permissions:**

- `extras.view_approvalworkflow` - to view created approval workflows after creating, updating, or running an object.
- `extras.view_approvalworkflowstage` - to view the **My Requests** tab on the Dashboard.
- `extras.view_approvalworkflowstageresponse` - to view responses for each stage.
```